pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/ContextImpl.java

```diff
@@ -300,6 +304,21 @@ public String getSecret(String secretName) {
         }
     }
 
+    @Override
+    public PulsarAdmin getPulsarAdmin() {
+        return getPulsarAdmin(defaultPulsarCluster);
+    }
+
+    @Override
+    public PulsarAdmin getPulsarAdmin(String clusterName) {
+        PulsarCluster pulsarCluster = externalPulsarClusters.get(clusterName);
+        if (pulsarCluster != null) {
+            return pulsarCluster.getAdminClient();
+        } else {
+            return null;
```

Comment:
shall we throw an error ? maybe reporting the list of known clusters ?
`
throw new IllegalArgumentException("PulsarAdmin for cluster "+clusterName+" is not available, only "+extrenalPulsarCluters.keySet())` 

if we do not throw an error, can we log something at warning level ?